unified-runtime/source/adapters/level_zero/v2/queue_create.cpp

```diff
@@ -12,9 +12,12 @@
 
 #include "logger/ur_logger.hpp"
 #include "queue_api.hpp"
+#include "queue_batched.hpp"
 #include "queue_handle.hpp"
 #include "queue_immediate_in_order.hpp"
 
+static const bool ForceBatched = getenv_tobool("UR_L0_FORCE_BATCHED");
```

Comment:
Should this option be documented in https://oneapi-src.github.io/unified-runtime/core/LEVEL_ZERO.html#environment-variables? Otherwise, a user would need to look into the code to find it.